playpen/vagrant-bootstrap-ansible.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+# minimal bootstrapping before kicking off ansible:
+#  install only what ansible needs to survive, or doesn't know how to do
+sudo dnf -y install python2 python2-dnf
```

Comment:
I'm not sure that dev-setup.sh needs to call this; I'll double check and make the appropriate changes.